frontends/mit-open/src/page-components/LearningResourceCardTemplate/LearningResourceCardTemplate.test.tsx

```diff
@@ -49,6 +50,7 @@ describe("LearningResourceCard", () => {
         suppressImage={true}
       />,
     )
+    console.log(images.length)
```

Comment:
Thanks for the context. I had this logging in here when I was fixing an issue with the test, but that ended up being due to an error in the way I implemented `imageSlot`. Anyway, that's fixed now. Anywho, this test is unchanged from the way it is on `main`: https://github.com/mitodl/mit-open/blob/88a87ec35fa6f1980b16dd871300293bd5d5b58c/frontends/mit-open/src/page-components/LearningResourceCardTemplate/LearningResourceCardTemplate.test.tsx#L33. Were you just pointing that out in case I was expecting to see a change between the two logging locations?

Reply:
> Were you just pointing that out in case I was expecting to see a change between the two logging locations?

yes